utils/date.tsx

```diff
@@ -0,0 +1,9 @@
+import {format, parseISO} from 'date-fns';
+
+export const dateToString = (dateString: string | undefined): string => {
```

Comment:
nit: let's call this `canonicalDateFormat` or something - lots of ways to `dateToString` ;)

Reply:
agreed that it should have a less generic name, but I wanted to wait until we add a second date format so that we could come up with names that are less generic. something tells me there won't be a single canonical format (there might be date vs date-and-time, for example).

Reply:
OK, let's start with `canonicalizeDateAndTime()` or `formatDateAndTime()` and iterate from there? We know even today it's doing more than `string` :)